discovery-provider/src/queries/get_health.py

```diff
@@ -702,3 +702,52 @@ def get_latest_chain_block_set_if_nx(redis=None, web3=None):
             )
 
     return latest_block_num, latest_block_hash
+
+
+def get_acdc_status() -> Tuple[Dict, bool]:
+    # setup return objects
+    res = {
+        "current_block": None,
+        "current_block_number": None,
+        "signers": None,
+        "signers_count": None,
+        "snapshot": None,
+        "me": None,
+        "is_signer": None,
+        "health": None,
+    }
+
+    # gather connections
+    web3endpoint = os.getenv("audius_web3_nethermind_rpc")
+    web3 = get_nethermind_web3(web3endpoint)
+
+    # web3 get current block
+    current_block = web3.eth.get_block('latest')
+    res["current_block"] = current_block
+    res["current_block_number"] = current_block.number
```

Comment:
want to just add signer info to the real health check?

Reply:
the main health check is already somewhat slow but we can move this over, was thinking we could poll from healthz and having a separate endpoint would be more performant

Reply:
i see, yea either way is good. this should be really fast though. health_check might be slow because of db queries.